src/nsls2api/models/slack_models.py

```diff
@@ -0,0 +1,21 @@
+import pydantic
+
+
+class SlackBot(pydantic.BaseModel):
+    username: str
+    user_id: str
+    bot_id: str
+
+
+class SlackUser(pydantic.BaseModel):
+    user_id: str
+    username: str
+    email: str
+
+
+class SlackChannelCreationResponseModel(pydantic.BaseModel):
+    channel_id: str | None = None
+    channel_name: str | None = None
+    beamline_slack_managers: list[str] | None = []
+    user_ids: list[str] | None = []
```

Comment:
Based on usage and validation in the API endpoint, I think it would be ok (perhaps better) to disallow null values for these fields.

```suggestion
    channel_id: str
    channel_name: str
    beamline_slack_managers: list[str] = []
    user_ids: list[str] = []
```

Reply:
I used `None` so it's easy to check if it's there or not and valid. For things that I depending on being a list I make the default a list.

Reply:
Looking closer, yes I agree for `channel_id` and `channel_name` - no sense to allow `None`.